### ISSUE TITLE
Use black text and icons on orange backgrounds

### ISSUE THREAD

**Issue by mxmou** [ScratchAddons/ScratchAddons#7357]
### Why this would be helpful

The orange accent color used on the settings page and in the popup doesn't have enough contrast with white.

### My suggestion

Change the color of white text and icons on an orange background to black.

### Possible alternatives

#6027

### Additional context

![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/f307045f-1a4c-4f86-a8f4-27271280fe2d)
(The actual extension icon doesn't need to change, but a white icon looks weird next to black text.)

**Comment by Secret-chest** [comment]
WCAG is broken.

**Comment by Secret-chest** [comment]
> Note: WCAG guidelines are known to have some issues. We shouldn't automatically determine against one set of rules, especially when brand color is at play.

https://www.bounteous.com/insights/2019/03/22/orange-you-accessible-mini-case-study-color-ratio

**Comment by Secret-chest** [comment]
![Screenshot_20240415_130814](https://github.com/ScratchAddons/ScratchAddons/assets/74449186/22be6369-2dcc-4694-959e-ddc701bf71fa)
![Screenshot_20240415_130736](https://github.com/ScratchAddons/ScratchAddons/assets/74449186/986476b9-2339-40e8-8521-e8a6b749470c)

The APCA shown here might be used in WCAG 3. Using black creates a vibration to me. We can change it to a darker colour, but no black, please.

**Comment by DNin01** [comment]
I think there's room for improvement with our current color contrast, but I believe current research shows that white text on color is preferable.

**Comment by BroJac5246** [comment]
> I believe current research shows that white text on color is preferable.

Do you have a source for this? My understanding is that it is generally preferable to present dark text on light backgrounds ("light mode"). I think part of the reason Maximouse's quick mock-up looks somewhat odd is because it's (at least to me) dark text on a dark background. With a lighter orange, it's plenty readable:
![Screenshot_20240430-154448.png](https://github.com/ScratchAddons/ScratchAddons/assets/104146239/bd9bdfd3-9016-4f75-b713-abba5a6ae1a9)

(I'm not able to create a mock-up right now so that'll have to do)

**Comment by DNin01** [comment]
> Do you have a source for this? My understanding is that it is generally preferable to present dark text on light backgrounds ("light mode"). I think part of the reason Maximouse's quick mock-up looks somewhat odd is because it's (at least to me) dark text on a dark background. With a lighter orange, it's plenty readable:

Yeah, black text is much better on light colors such as yellow.

**Comment by Secret-chest** [comment]
We could consider completely changing it.

**Comment by Secret-chest** [comment]
> Do you have a source for this? My understanding is that it is generally preferable to present dark text on light backgrounds ("light mode"). I think part of the reason Maximouse's quick mock-up looks somewhat odd is because it's (at least to me) dark text on a dark background. With a lighter orange, it's plenty readable:

That is light, not dark. Our current orange can be considered dark.

**Comment by Secret-chest** [comment]
![Screenshot_20240501_135818](https://github.com/ScratchAddons/ScratchAddons/assets/74449186/f7cb4f07-159f-46bb-b6a6-85b4dadd0173)
An orange that works with black

**Comment by mxmou** [comment]
I'm closing this because it's a bad idea. We should find a different way to fix the contrast - selected buttons could be blue instead of orange, for example:

<img width="309" height="68" alt="Image" src="https://github.com/user-attachments/assets/3580d667-ddb6-4b47-93c7-38ef36ad4d58" /><br />

<img width="309" height="58" alt="Image" src="https://github.com/user-attachments/assets/505bea25-302c-443e-85e6-918b889c953f" />

We could also do something similar to the popup tabs:

<img width="658" height="66" alt="Image" src="https://github.com/user-attachments/assets/3f9eef5d-078f-41eb-a134-b85513b85700" /><br />

<img width="655" height="56" alt="Image" src="https://github.com/user-attachments/assets/96419c2b-4d6b-40c9-b8f8-c3a828b031d3" />

**Comment by DNin01** [comment]
I also tried adding a shadow, an often-overlooked way to boost text contrast.

![A select setting with a shadow under the text on the selected item](https://github.com/user-attachments/assets/a0daaada-1f33-48b1-a02f-7e4746625175)
